### PR TITLE
add flux ecosystem event blog post

### PR DESCRIPTION
**Pull request by dholbach**
Signed-off-by: Daniel Holbach <daniel@weave.works>

### CODE REVIEW

**Comment by dholbach** [comment]
Review welcome - not sure why the link checker is failing - will likely get another small update on Monday.

File: content/en/blog/2021-09-17-gitops-one-stop-shop-event.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+url: /blog/2021/09/gitops-one-stop-shop/
+images:
+- /img/2021-09-gitops-one-stop-shop-event.jpg
+---
+
+![](/img/2021-09-gitops-one-stop-shop-event.jpg)
+
+Flux is going to be center stage at the [GitOps One Stop Shop Event](https://gitopsdays.com) on Oct 20th, 2021.
+
+Our friends over at GitOps Days will be hosting a special community edition to invite vendors who have chosen to back their GitOps offering by Flux. We are very excited that the ecosystem around GitOps is growing and so many chose Flux to bring the goodness we believe in to their users and customers.
```
Comment (scottrigby) [inline content/en/blog/2021-09-17-gitops-one-stop-shop-event.md:16]: Is this fair to say? Are these vendors all "cloud providers"?

```suggestion
Our friends over at GitOps Days will be hosting a special community edition to invite cloud providers who have chosen to back their GitOps offering by Flux. We are very excited that the ecosystem around GitOps is growing and so many chose Flux to bring the goodness we believe in to their users and customers.
```

Comment (dholbach) [reply content/en/blog/2021-09-17-gitops-one-stop-shop-event.md:16]: AFAICS not everyone who is presenting is a cloud provider.

Comment (makkes) [reply content/en/blog/2021-09-17-gitops-one-stop-shop-event.md:16]: Yes, D2IQ is not a cloud provider so vendor seems to be a better, more general, fit for both, product and cloud vendors.

File: content/en/blog/2021-09-17-gitops-one-stop-shop-event.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+What does this mean for you? If you are considering GitOps for your company and want to see what these vendors have to offer, this is your one-stop shopping experience for GitOps! Hear from Amazon, D2iQ, Microsoft, VMware, and Weaveworks on their GitOps offerings.
+
+Showcased products are:
+
+- **Amazon** EKS Anywhere
+- **D2iQ** DKP Dispatch
```
Comment (makkes) [inline content/en/blog/2021-09-17-gitops-one-stop-shop-event.md:23]:
```suggestion
- **D2iQ** DKP Kommander
```

Dispatch functionality is being rolled into Kommander.

**Review by staceypotter — COMMENTED** [review]
Just the one typo fix from me, otherwise LGMT! 

Thanks @dholbach

**Review by alisondy — APPROVED** [review]
lgtm

**Pull request merged**